File: src/main/java/eu/karelhovorka/zpevnik/music/ChordType.java
package eu.karelhovorka.zpevnik.music;

import java.util.List;

import eu.karelhovorka.zpevnik.util.Tone;

public enum ChordType {
	DUR(new String[]{"", "dur"}, Interval.MAJOR_THIRD.step, Interval.PERFECT_FIFTH.step),
	MOL(new String[]{"mol", "mi", "m"}, Interval.MAJOR_THIRD.step, Interval.PERFECT_FIFTH.step);
	
	
	private ChordType(String[] aliases, int... intervals) {
		
	}
	
	private List<Integer> intervals;
}

File: src/main/kotlin/eu/karelhovorka/zpevnik/music/ChordType.kt
package eu.karelhovorka.zpevnik.music

enum class ChordType private constructor(aliases: Array<String>, vararg intervals: Int) {
    DUR(arrayOf("", "dur"), Interval.MAJOR_THIRD.step, Interval.PERFECT_FIFTH.step),
    MOL(arrayOf("mol", "mi", "m"), Interval.MAJOR_THIRD.step, Interval.PERFECT_FIFTH.step);

    private val intervals: List<Int>? = null
}
